app/api/courses/courses.interface.ts

```diff
@@ -8,7 +8,7 @@ export interface CourseProps {
   image_url: string;
   description: string;
   created_at: string;
-  duration: number;
+  duration: number; // in seconds
```

Comment:
```suggestion
  /*
  *  This approach will put current comment to docs and it will be available as part of signature in your IDE
  */
  duration: number; 
```